x/marker/types/msg_test.go

```diff
@@ -689,3 +689,94 @@ func TestMsgSetAccountDataRequestGetSigners(t *testing.T) {
 		require.PanicsWithError(t, "decoding bech32 failed: invalid separator index -1", testFunc, "GetSigners")
 	})
 }
+
+func TestMsgUpdateSendDenyListRequest(t *testing.T) {
```

Comment:
Nit 2: This function should be called `TestMsgUpdateSendDenyListRequestValidateBasic` since it's only testing the `ValidateBasic` method.